### DISCUSSION TITLE
Problem show image

### DISCUSSION THREAD

**Discussion by poncho-ajmv** [catsout/wallpaper-engine-kde-plugin#287] [Q&A]
I have all the dependencies installed but it appears that it is not in the image, I look for the folder as they are in the instructions, but it does not work, it does not detect anything.
![Screenshot_20230324_215515](https://user-images.githubusercontent.com/98699599/227691960-f0abfd31-872c-4228-8174-54626ac292bf.png)
![Screenshot_20230324_215519](https://user-images.githubusercontent.com/98699599/227691968-1cdcedc6-6c0b-47ed-8c6d-222c50a01c08.png)

**Comment by Willlost** [comment]
your wallpaper engine must be installed in the default location where your steam folder is at.
